nrf_security/Kconfig

```diff
@@ -585,57 +584,42 @@ config VANILLA_MBEDTLS_CMAC_C
 
 if NRF_SECURITY_MULTI_BACKEND
 
-config GLUE_MBEDTLS_CMAC_C
-	bool
-	depends on GLUE_MBEDTLS_AES_C
-	default y
-
-config GLUE_CC3XX_MBEDTLS_CMAC_C
-	bool
-	depends on CC3XX_MBEDTLS_AES_C
-	depends on GLUE_MBEDTLS_AES_C
-	default y
-
-config GLUE_OBERON_MBEDTLS_CMAC_C
-	bool
-	depends on OBERON_MBEDTLS_AES_C
-	depends on GLUE_MBEDTLS_AES_C
-	default y
-
-config GLUE_VANILLA_MBEDTLS_CMAC_C
+choice
 	bool
-	depends on VANILLA_MBEDTLS_AES_C
-	depends on GLUE_MBEDTLS_AES_C
-	default y
+	prompt "Backend selection"
+	depends on 	NRF_SECURITY_MULTI_BACKEND
```

Comment:
Why the extra spacing ?

Reply:
Tab failure, will be fixed. We are still running internal tests before we are taking this for CI, BTW. 
Hence no nrf/manifest yet

Reply:
Fixed